lib/browser.js

```diff
@@ -1,3 +1,11 @@
+var util = require('./util');
```

Comment:
@chrisradek Do you think it might be worthwhile adding a buffer polyfill as well?